crypt/aes.go

```diff
@@ -0,0 +1,84 @@
+package crypt
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"io"
+
+	"github.com/goravel/framework/facades"
+)
+
+type AES struct {
+	key []byte
+}
+
+// NewAES returns a new AES hasher.
+func NewAES() *AES {
+	key := facades.Config.GetString("app.key")
+	// check key length before using it
+	if len(key) != 16 && len(key) != 24 && len(key) != 32 {
+		panic("[crypt] app.key length must be 16, 24 or 32")
+	}
+	keyBytes := []byte(key)
+	return &AES{
+		key: keyBytes,
+	}
+}
+
+// EncryptString encrypts the given string, and returns the iv and ciphertext as base64 encoded strings.
+func (b *AES) EncryptString(value string) (string, string) {
+	block, err := aes.NewCipher(b.key)
+	if err != nil {
+		panic(err.Error())
```

Comment:
我们一般不 panic，可以改为（下同）：
```suggestion
		return "", err
```